roles/multus-crd/templates/macvlan.yml.j2

```diff
@@ -0,0 +1,21 @@
+apiVersion: "kubernetes.com/v1"
```

Comment:
This template has a lot of hard coded values :-1:

Reply:
good eye, copy-pasta from a generated template, will match original multus template with vars from all.yaml